server/container/core/pom.xml

```diff
@@ -77,6 +89,18 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>1.7.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>pl.pragmatists</groupId>
```

Comment:
I think this is needed because the test-jar don't get the dependency. But the version should be in a parent POM